doc/whats_new/v1.0.rst

```diff
@@ -623,6 +623,9 @@ Changelog
   amount of data.
   :pr:`20312` by :user:`Divyanshu Deoli <divyanshudeoli>`.
 
+- |Fix| Fix recall in multilabel classification when true labels are all negative.
+  :pr:`19085` by :user:`Varun Agrawal <varunagrawal>`.
```

Comment:
We can adjust the whats new to state which function is being fixed.

```rst
- |Fix| Fixes `average_precision_score` for multilabel classification when true labels are
   all negative. :pr:`19085` by :user:`Varun Agrawal <varunagrawal>`.
```

There also needs to be an entry for `precision_recall_curve` to describe the new behavior.

These entries need to be moved to `doc/whats_new/v1.1.rst`.